web/pages/contract-search-firestore.tsx

```diff
@@ -21,10 +16,10 @@ export default function ContractSearchFirestore(props: {
   }
 }) {
   const contracts = useContracts()
-  const { querySortOptions, additionalFilter } = props
+  const { additionalFilter } = props
 
-  const { query, setQuery, sort, setSort } =
-    useQueryAndSortParams(querySortOptions)
+  const [query, setQuery] = useQuery('', true)
```

Comment:
Not a big fan of boolean arguments because as a reader you have to check the definition to see what it's doing.

An alternative is an `options` object, e.g.

```
useQuery(defaultQuery: string, options: { useUrl: boolean })
```

Reply:
OK, I am kind of indifferent but I understand your POV so I will change it.